.github/workflows/android.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://docs.github.com/en/actions/guides/building-and-testing-java-with-gradle
+
+name: CI Android
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: JDK ${{ matrix.java_version }}
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        # TODO Add 9, 10, 11, 12, and 13 after Kotlin 1.3.60
+        java_version: [1.8]
+    steps:
+    - name: Checkout
+    - uses: actions/checkout@v2
+    - name: Install JDK
+      uses: actions/setup-java@v2
+      with:
+        java-version: ${{ matrix.java_version }}
+    - name: Gradle Wrapper Validation
+      uses: gradle/wrapper-validation-action@v1
+    - name: Configure Gradle
+      # Gradle configuration install deps 
+      run: ./gradlew help
+    - name: Spot check
+      run: ./gradlew spotlessCheck --stacktrace
+    - name: Build Project
+      run: ./gradlew assemble --stacktrace
```

Comment:
is it better to split these steps out into assemble/test/check or just used `build` like [travis setup](https://github.com/uber/RIBs/blob/master/.travis.yml#L24) did?

Reply:
This likely gives better output and allows for things like paralllelizing. See https://github.com/uber/AutoDispose/blob/main/.github/workflows/ci.yml